css-three-column-layout/styles.css

```diff
@@ -0,0 +1,56 @@
+* {
+  font-family: Lato, serif;
+  background-color: rgb(25, 150, 218);
+  color: white;
+}
+
+section {
+  max-width: 2004px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+h2 {
+  font-size: 62px;
+  margin-top: 120px;
+  letter-spacing: 0.92px;
+  font-weight: 300;
+}
+
+h3 {
+  font-size: 43px;
+  font-weight: 300;
+}
+
+p {
+  font-size: 29px;
+  margin: 0 80px;
+  color: rgb(221, 221, 221);
+}
+
+.white-circle {
+  background-color: white;
+  width: 240px;
+  height: 240px;
+  border: 2px solid rgb(221, 221, 221);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 120px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+img {
+  background-color: white;
+  width: calc(124% / 2.4);
+}
```

Comment:
This is a bit of a weird way to do it. While it may work in appearance, this would be hard to read code since it goes into a more difficult and you're doing this for 4px (ends up being 124px). Just make this a width of 50% it's parent.

You are also adding a `background-color: white` here, but you have its parent with the `.white-circle` which has a `background-color: white` too. Is this necessary here?

Reply:
@width: alright, I suppose weird calculations should be kept to a minimum.
@bg-color: it was necessary when I had everything set to bg-color ~blue; I switched it to .body that now has a bg-color of ~blue, so the img background is no longer necessary.

Reply:
*whoops! I meant body the type selector, not .body a class selector